known_drivers/mk-nvidia-drivers.py

```diff
@@ -0,0 +1,69 @@
+import requests
+from bs4 import BeautifulSoup
+import re
+import subprocess
+from subprocess import CalledProcessError
+import os
+import shutil
+
+CLEANUP=True
+
+headers = {'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/39.0.2171.95 Safari/537.36'}
+
+nv = requests.get("https://download.nvidia.com/XFree86/Linux-x86_64/", headers=headers)
+
+soup = BeautifulSoup(nv.content.decode(), 'html.parser')
+dirs = [e.text for e in soup.select(".dir")]
+vers = [re.sub("/$", "", t) for t in dirs if not re.search("^..$|-", t)]
+
+def process_output(o):
+    lines = o.decode().split("\n")
+    pth_line = lines[0]
+    store_pth = re.search("'(.*)'", pth_line).group(1)
+    store_hash = lines[1]
+
+    return (store_pth, store_hash)
+
+if os.path.exists("driver-versions.nix"):
+    print("driver-versions.nix already exists, parsing known versions to skip re-downloads.")
+    print("copying old versions to driver-versions.bak.nix.")
+    shutil.copyfile("driver-versions.nix", "driver-versions.bak.nix")
```

Comment:
We are using git, so this is not required

Reply:
good point, will remove